crates/pop-parachains/src/build/mod.rs

```diff
@@ -326,6 +329,26 @@ impl ChainSpec {
 		fs::write(path, self.to_string()?)?;
 		Ok(())
 	}
+
+	/// Updates the runtime code in the chain specification.
+	///
+	/// # Arguments
+	/// * `wasm_bytes` - The new runtime Wasm code.
+	pub fn update_runtime_code(&mut self, wasm_bytes: &[u8]) -> Result<(), Error> {
+		let wasm_hex = to_hex(wasm_bytes, true);
```

Comment:
```suggestion
		let hex = to_hex(bytes, true);
```

Probably better to only do this conversion once the json value has been successfully resolved.

Reply:
https://github.com/r0gue-io/pop-cli/pull/425/commits/c7dea912f7efe5ac15c68d47e13e6dfc6c2b85cc